### ISSUE TITLE
global service is unsuport 

### ISSUE THREAD

**Issue by dxinli** [jetpack-io/devbox#1480]
**Current Behavior (bug)**
i add global mysql , but i can not using `devbox global service up` in a devbox project,as before, global package is available

**Expected Behavior (fix)**
This command replicates the subcommands for [devbox services] but for your global packages,otherwise ,is not

**Additional context**
version is 0.5.13
global package devbox.json :
{
  "packages": [
    "redis@latest",
    "postgresql@latest",
    "mysql80@latest"
  ],
  "shell": {
    "init_hook": [
      "echo 'Welcome to devbox!' > /dev/null"
    ],
    "scripts": {
      "test": [
        "echo \"Error: no test specified\" && exit 1"
      ]
    }
  }
}

**Comment by Lagoja** [comment]
I'm seeing the following error: 
```
Error: No devbox.json found in this directory, or any parent directories. Did you run `devbox init` yet?
```
It looks like an issue with how we detect the devbox.json. I'll take a look into what's happening

**Comment by dxinli** [comment]
yeah, poject is inited, but devbox global services is not work

**Comment by dxinli** [comment]
`devbox global ls` , output:
* redis@latest
* postgresql@latest
* mysql80@latest
but `devbox gloabal services ls` ,output:
No services found in your project
doc for `devbox global services ls` is "Interact with Devbox services for your global packages. This command replicates the subcommands for but for your global packages.", is not right

**Comment by Lagoja** [comment]
@dxinli we've identified a bug that is causing this, and it should be fixed with #1484. This change should go out in our next release.

**Comment by dxinli** [comment]
ok , thanks

**Comment by Lagoja** [comment]
This fix #1484 should now be merged in 0.5.14. Please let us know if it works for you!
